iwyu_ast_util.cc

```diff
@@ -556,24 +557,29 @@ class TypeEnumerator : public RecursiveASTVisitor<TypeEnumerator> {
 
   set<const Type*> Enumerate(const TemplateArgument& tpl_arg) {
     seen_types_.clear();
-    TraverseTemplateArgument(tpl_arg);
+    this->TraverseTemplateArgument(tpl_arg);
```

Comment:
I'm guessing `this->` qualification is not necessary here, since you don't have it for `TraverseType` above.

Reply:
Neither MSVC nor GCC compiles this without `this->`. The difference with `TraverseType` is that the called overload of the latter is declared directly in `TypeEnumerator` and doesn't come from the templated base.

Reply:
Aha, I see, thanks